IntelFsp2Pkg/Include/FspEas/FspApi.h

```diff
@@ -296,10 +435,35 @@ typedef struct {
 /// Action definition for FspMultiPhaseSiInit API
 ///
 typedef enum {
-  EnumMultiPhaseGetNumberOfPhases  = 0x0,
-  EnumMultiPhaseExecutePhase       = 0x1
+  EnumMultiPhaseGetNumberOfPhases       = 0x0,
+  EnumMultiPhaseExecutePhase            = 0x1,
+  EnumMultiPhaseGetVariableRequestInfo  = 0x2,
+  EnumMultiPhaseCompleteVariableRequest = 0x3
 } FSP_MULTI_PHASE_ACTION;
 
+typedef enum {
+ EnumFspVariableRequestGetVariable          = 0x0,
+ EnumFspVariableRequestSetVariable          = 0x1,
+ EnumFspVariableRequestGetNextVariableName  = 0x2,
```

Comment:
Enum 0x1 and 0x2 seem to be swapped compared to what is defined in FSP spec/FSP code.

Reply:
Good catch. I think this must have been an accidental drag/drop after testing.